Update windows-enrollment-status.md (#4007)
MicrosoftDocs/memdocs: memdocs/intune/enrollment/windows-enrollment-status.md

 Intune applies profiles in the following order:  

> 1. Intune applies the highest-priority profile assigned to the device (exluding the default ESP profile). 

Reviewer: ```suggestion
1. Intune applies the highest-priority profile assigned to the device (excluding the default ESP profile). 
```

> 2. If no profiles are targeted at the device (exluding the default ESP profile), Intune applies the highest-priority profile assigned to the user. This only works in scenarios where there is a user. In white glove and self-deploying scenarios, only profiles targeted at devices can be applied.   

Reviewer: ```suggestion
2. If no profiles are targeted at the device (excluding the default ESP profile), Intune applies the highest-priority profile assigned to the user. This only works in scenarios where there is a user. Only profiles targeted at devices can be applied in white glove and self-deploying scenarios.   
```

> 2. If no profiles are targeted at the device (excluding the default ESP profile), Intune applies the highest-priority profile assigned to the user. This only works in scenarios where there is a user. Only profiles targeted at devices can be applied in white glove and self-deploying scenarios.   

Reviewer: ```suggestion
2. If no profiles are targeted at the device (excluding the default ESP profile), Intune applies the highest-priority profile assigned to the user. This only works in scenarios where there is a user. Only profiles targeted at devices can be applied in pre-provisioning and self-deploying scenarios.   
```
